src/patternfly/components/Divider/examples/Divider.md

```diff
@@ -7,64 +7,88 @@ cssPrefix: pf-c-divider
 import './Divider.css'
 
 ## Examples
+
 ### hr
+
 ```hbs
-{{> divider}}
+{{> divider divider--modifier="pf-m-horizontal"}}
```

Comment:
I don't think we want to use `.pf-m-horizontal` on its own anywhere since it isn't necessary. We also use the divider throughout the site in other components/demos that wouldn't have `.pf-m-horizontal`, so it could be confusing.

You could even argue excluding `.pf-m-horizontal` as a class on its own by wrapping this line in a conditional that only includes the media query code if `($breakpoint != "base")` https://github.com/patternfly/patternfly/blob/7ab7699daf915fdb158dd78c5af38837daa38f6b/src/patternfly/components/Divider/divider.scss#L87-L90

Reply:
Sounds good - updated!